### ISSUE TITLE
Camera videos are rendered in grayscale in Chromium 98/ Windows

### ISSUE THREAD

**Issue by xuesichao** [aws/amazon-chime-sdk-component-library-react#742]
### What happened and what did you expect to happen?

Camera videos are rendered in grayscale on Chrome 98/ Windows when 5-8 videos are enabled in a call and hardware acceleration is enabled.

#### Cause

The issue occurs on the video sender side with the default video uplink bandwidth policy. If the video sender is using Chromium 98 on Windows and there are 5-8 videos enabled in the meeting. The video source from this sender will turns into grayscale, and other attendees can observe it.

The `NScaleVideoUplinkBandwidthPolicy` downgrades the video resolution as the number of videos increases. When there are 5-8 videos enabled in a call, the video resolutions are downgrade to 480P. At this moment the videos are rendered in grayscale. The encoder seems to be broken in Chrome 98 when encoding the videos at 480P.

### Have you reviewed our existing documentation?

- [X] Amazon Chime SDK for JavaScript GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] Amazon Chime SDK React Components Library GitHub [issues](https://github.com/aws/amazon-chime-sdk-component-library-react/issues)
- [X] Storybook [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/introduction--page)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-component-library-react#readme)
- [X] How-to [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/how-tos--page)
- [X] React SDK meeting [demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/meeting#readme---react-meeting-demo)

### Reproduction steps
* Enable hardware acceleration
* 5 attendees join a call and enable the camera.
* When the 5th attendee enable the camera, the video tiles are rendered in grayscale.

### Amazon Chime SDK React Components Library version

N/A

### What browsers are you seeing the problem on?

Chromium-based

### Browser version

98

### Device Information

N/A

### Browser console logs

N/A

**Comment by xuesichao** [comment]
Check more details here: https://github.com/aws/amazon-chime-sdk-js/issues/2002

**Comment by xuesichao** [comment]
We will release a hotfix in `amazon-chime-sdk-js` soon. 
To fix this issue in React Component Library, you will need to run upgrade the version of `amazon-chime-sdk-js`.

```
npm upgrade amazon-chime-sdk-js
```

**Comment by xuesichao** [comment]
We are working on a hotfix in `amazon-chime-sdk-js` now.

To fix this issue in React Component Library, you will need to run upgrade the version of `amazon-chime-sdk-js`.

```
npm upgrade amazon-chime-sdk-js
```

Will post the update here once we release the hotfix in `amazon-chime-sdk-js`.

**Comment by xuesichao** [comment]
Hi, we have published the hotfix `amazon-chime-sdk-js@2.26.1` to npm. If you are having this issue, could you please upgrade the `amazon-chime-sdk-js` in your project and let us know the result?
```
npm upgrade amazon-chime-sdk-js
```

**Comment by xuesichao** [comment]
Close the issue as fix has been published in `amazon-chime-sdk-js@2.26.1`.
